www/docs/style/sass/pages/_search.scss

```diff
@@ -449,10 +449,226 @@ table.search-results-grouped {
   }
 }
 
-.search-alert-type--rss {
+.sidebar-item-with-icon--rss {
```

Comment:
Also, it looks like search-alert-type and *--rss are still used in search/sidebar.php.